identity-core/src/crypto/key/key.rs

```diff
@@ -88,3 +88,9 @@ impl<V: private::Sealed> From<Vec<u8>> for Key<V> {
     other.into_boxed_slice().into()
   }
 }
+
+impl<V: private::Sealed> From<&Key<V>> for Vec<u8> {
+  fn from(other: &Key<V>) -> Self {
+    other.as_ref().to_vec()
+  }
+}
```

Comment:
This seems unnecessary, can't one just call `as_ref` to access the underlying `&[u8]` then call `to_vec` on that?

Reply:
We can. I just thought calling `into()` would be more convenient, but I'll remove this.